politeiawww/api/cms/v1/v1.go

```diff
@@ -992,3 +994,35 @@ type CastVoteReply struct {
 	Error           string                 `json:"error"`                 // Error status message
 	ErrorStatus     cmsplugin.ErrorStatusT `json:"errorstatus,omitempty"` // Error status code
 }
+
+// ProposalBillingSummary allows for all proposal spending to be returned for
+// an admin to review.
+type ProposalBillingSummary struct {
```

Comment:
This call is going to need to be paginated at some point but I think its fine for now to return all.

Reply:
I agree and think this should be done before this goes in. A simple `offset` and `count` should suffice.